### DISCUSSION TITLE
Should we add more gravity to sam?

### DISCUSSION THREAD

**Discussion by LUIS9799** [pege22/pege_source_main#20] [Ideas]
Actually sam is jumping with too much force, should we change jump force and gravity, I just tested it and it gone well, sam can get over all levels and can jump to each block normally.

**Comment by JayOr209** [comment]
i think no

**Reply by LUIS9799** [reply]
Well the thing is that it makes too easy for players to cheat on the game hehe

**Comment by LUIS9799** [comment]
Fixed
